apiserver/service/router.go

```diff
@@ -68,6 +68,9 @@ func NewServerAndRun(conf config.ServerConfig) {
 	chatGroup := r.Group("/chat")
 	registerChat(chatGroup, conf)
 
+	fg := r.Group("/forward")
+	registerForward(fg)
```

Comment:
do we have any auth check(authentication or authorization) for this router?

Reply:
I don’t think it’s necessary, we’re just forwarding it.

Reply:
we can only check whether the user have valid platform token,then we can track the api call .